src/app/app.module.ts

```diff
@@ -87,6 +89,8 @@ import { MatSnackBarModule } from '@angular/material/snack-bar';
     MatTabsModule,
     MatDialogModule,
     RouterModule,
+    MatDividerModule,
```

Comment:
think that we should import material related modules in neo-{name} related module